tools/osx/crosstool/cc_toolchain_config.bzl

```diff
@@ -93,6 +93,10 @@ def _impl(ctx):
         target_system_name = "arm64-apple-macosx{}".format(target_os_version)
     elif (ctx.attr.cpu == "darwin_arm64e"):
         target_system_name = "arm64e-apple-macosx{}".format(target_os_version)
+    elif (ctx.attr.cpu == "maccatalyst_x86_64"):
+        target_system_name = "x86_64-apple-ios{}-macabi".format(target_os_version)
```

Comment:
I think this is for clang?

For example,

```shell
clang -target arm64-apple-ios13.1-macabi catalyst_main.c
```

Reply:
Yep looks right, it's used for clang, swift, and the linker.